modules/API/pages/gsql-endpoints.adoc

```diff
@@ -0,0 +1,6093 @@
+Sample Request::
++
+--
+[source,bash]
+----
+curl -X DELETE "Content-type: application/json" "http://localhost:14240/gsql/scim/v2/Groups/g1?gsqlFormat=false" -d '
+{"schemas":"[\"urn:ietf:params:scim:api:messages:2.0:PatchOp\"]","Operations":[{"op":"remove","path":"members"},{"op":"replace","path":"members","value":[{"display":"user4"}]},{"op":"add","value":[{"display":"user3"}]}]}'
+----
+--
+Sample Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"Successfully dropped group g1"}
+----
+--
+====
+
+'''
+
+=== drop groups
+
+`POST /gsql/scim/v2/Groups`
+
+This endpoint is used to drop groups.
+
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `action`
+| no
+| The default value is `create`. Possible values are `create` and `delete`. `Delete` indicates dropping groups, while `create`  indicates creating groups.
+|===
+
+==== Example:
+[,tabs]
+====
+Sample Request::
++
+--
+[source,bash]
+----
+curl -X POST "Content-type: application/json" "http://localhost:14240/gsql/scim/v2/Groups?action=delete" -d '{"groupNames":["g1"]}'
+----
+--
+Sample Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"Successfully dropped groups: [g1]"}
+----
+--
+====
+
+'''
+
+=== retrieve a user by id using scim/gsql format
+
+`GET /gsql/scim/v2/Users/{id}`
+
+This endpoint is used to retrieve a user by id respond with scim or gsql format.
+
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `gsqlFormat`
+| no
+| By default, the response is in SCIM formatted JSON (default: `false`). Setting it to `true` will respond in GSQL formatted JSON.
+|===
+
+==== Example:
+[,tabs]
+====
+Sample Request::
++
+--
+[source,bash]
+----
+curl -X GET "http://localhost:14240/gsql/scim/v2/Users/u1?gsqlFormat=false"
+----
+--
+Sample Response::
++
+--
+[source.wrap,console]
+----
+{"meta":{"location":"/scim/v2/Users/1e224fd0-54eb-43c3-84d6-455ca3b6339d","resourceType":"User"},"schemas":["urn:ietf:params:scim:schemas:core:2.0:User"],"name":{},"active":true,"id":"1e224fd0-54eb-43c3-84d6-455ca3b6339d","userName":"u1"}
+----
+--
+====
+
+'''
+=== update a user
+
+`PUT /gsql/scim/v2/Users/{id}`
+
+This endpoint is used to update a user.
+
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `gsqlFormat`
+| no
+| The default value is `false`. When set to `false`, the request body uses the SCIM format JSON; when set to `true`, the request body uses the gsql format JSON.
+|===
+
+==== Example:
+[,tabs]
+====
+Sample Request::
++
+--
+[source,bash]
+----
+curl -X PUT -H "Content-type: application/json" "http://localhost:14240/gsql/scim/v2/Users/u1?gsqlFormat=false"  -d '{"schemas":"[\"urn:ietf:params:scim:api:messages:2.0:PatchOp\"]","Operations":[{"op":"replace","path":"password","value":"newPassword"}]}'

```

Comment:
remove `gsqlFormat=false` since the default is false, and similar for all others